tools/tests/test_osqueryd_win.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python
+
+#  Copyright (c) 2014-present, Facebook, Inc.
+#  All rights reserved.
+#
+#  This source code is licensed under the BSD-style license found in the
+#  LICENSE file in the root directory of this source tree. An additional grant
+#  of patent rights can be found in the PATENTS file in the same directory.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import random
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+import unittest
+
+CONFIG_FILE = """
+{
+  "options": {
+  },
+
+  "schedule": {
+    "processes_test": {
+      "query": "select pid, name, path, cmdline from processes",
+      "interval": 180
+    }
+  }
+}
+"""
+
+
+def assertUserIsAdmin():
+    if os.name != "nt":
+        sys.exit(-1)
+    try:
+        os.listdir("\\Windows\\Temp")
+    except WindowsError:
+        sys.exit(-1)
+
+
+def sc(*args):
+    try:
+        output = subprocess.check_output(["sc.exe"] + list(args))
```

Comment:
Any chance you could grab stdout and stderr here and print them in the exception? Either that or grab the exception and display what it its? Might be nice for debugging if something starts going wrong?